docs/release-process.md

```diff
@@ -0,0 +1,129 @@
+# samba-container Release Process
+
+## Preparation
+
+The samba-container project has a dedicated branch, called `release`, for
+release versions. This is done to update certain files which control
+dependencies and versioning. Tags are applied directly to this branch
+and only this branch.
+
+
+### Tagging
+
+Prior to tagging, check out the `release` branch and merge `master` into it.
+Example:
+
+```
+git checkout master
+git pull --ff-only
+git checkout release
+git pull --ff-only
+git merge master
+# resolve any conflicts
+```
+
+Now we need to "pin" the appropriate versions of samba and sambacc so that only
+explicitly specified versions of those packages will be included on release
+branch builds. Set `SAMBA_VERSION_SUFFIX` and `SAMBACC_VERSION_SUFFIX` in the
+container files in `images/`. Currently, only the fedora based images are
+included in the release. Thus you must set those variables in the fedora
+container files for server and ad-server. Commit these changes to the `release`
```

Comment:
Does this mean to push changes directly instead of using pull requests? Ig yes, I think it would be helpful to mention this explicitly ...

Reply:
Added & expanded upon.